basic/list1.py

```diff
@@ -33,8 +36,18 @@ def match_ends(words):
 # Hint: this can be done by making 2 lists and sorting each of them
 # before combining them.
 def front_x(words):
-    # +++your code here+++
-    return
+    new=sorted(words)
```

Comment:
you seem to have went full C programmer :)

overall it is OK, but could be improved a great deal by not using indexes for iterating over a collection. It is pretty complex code this way.